### PR TITLE
add clipboard icon to linter messages

### LINKED ISSUE #78: allow highlighting and copying from the lint popups
Users like to copy paste error messages to google to find aid. It would be great if there were one or many of the following options:

1. a small button for copying the message to the clipboard (like on github with the repository urls)
2. text can be highlighted and copied in the messages
3. a button that goes straight to your browser and searches for the text (my least favorite option, since users will want to configure it)

### PR DESCRIPTION
**Pull request by oli-obk**
fixes #78 

screenshot from 2016-05-18 13-44-57

### CODE REVIEW

File: lib/elements/message.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ export default class Message extends React.Component {
     linkContent += ` in ${atom.project.relativizePath(message.filePath)[1]}`
     return <a href="#" className="linter-message-link" onClick={() => visitMessage(message) }>{ linkContent }</a>
   }
+  static getCopyLink(message: Linter$Message) {
+    return <a href="#" className="linter-message-copy-link" onClick={() => atom.clipboard.write(message.text) } />
```
Comment (steelbrain) [inline lib/elements/message.js]: This only supports `message.text`, what about `message.html`?

Comment (oli-obk) [inline lib/elements/message.js]: Good point. Iirc it's illegal to have both, so `message.text || message.html` should be ok?

Comment (steelbrain) [inline lib/elements/message.js]: Yeah sure

**Comment by oli-obk** [comment]
updated

**Comment by steelbrain** [comment]
Thanks 👍 I might an option to toggle this behavior in the future

**Pull request merged**